src/components/features/Deals/Deals.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './Deals.module.scss';
+
+class Deals extends React.Component {
+  render() {
+    const { products } = this.props;
+
+    const dealsProducts = products.filter(item => item.deals === true);
+
+    return (
+      <div className={styles.root}>
+        <div className={styles.component}>
+          <div className='container'>
+            <div className={`row ${styles.dealsRow}`}>
+              {dealsProducts.map(item => (
```

Comment:
Na grafice jest jeden duży i 2 małe, więc to musi być rozróznione w JSX

Reply:
poprawiłem, wyszedłem z założenia, że to tylko statyczny kompopnent...